cmd/bolt/main.go

```diff
@@ -146,7 +146,7 @@ The commands are:
     buckets     print a list of buckets
     check       verifies integrity of bolt database
     compact     copies a bolt database, compacting it in the process
-    dump        print a hexidecimal dump of a single page
+    dump        print a hexadecimal dump of a single page
```

Comment:
https://github.com/boltdb/bolt/commit/4660afd75e9f684dd589a857a0103814952560b0 didn't catch this oops